ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -345,10 +346,23 @@ static int HandleVnodeOperation(
         
         // Call vn_getpath first when the cache is hottest to increase the chances
         // of successfully getting the path
-        if (0 == vn_getpath(currentVnode, vnodePathBuffer, &vnodePathLength))
+        int error = vn_getpath(currentVnode, vnodePathBuffer, &vnodePathLength);
```

Comment:
Thoughts on 'result' or 'returnValue' instead of 'error'.

Reply:
I guess I picked "error" because this function returns `errno` style errors (`EINVAL`, `ENOSPC`, etc.). There's also no symbolic "success" constant for this type of result, so the check for that case, `(0 == error)` perhaps is a little clearer ("no error occurred") than `(0 == result)`.

Incidentally, I'll replace the declaration as `int` with `errno_t` to make the pool of possible cases clearer.